test/test_client_drag.res

```diff
@@ -67,7 +67,11 @@ let onMouseDown = onCB("mousedown", "", ev =>
   Json.Decoder.decodeEvent(
     Json.Decoder.map(dragStart, Mouse.position),
     ev,
-  ) |> result_to_option
+  ) |>  x =>
+  switch x {
```

Comment:
It may be best to keep result_to_option in a tea_result as well, rather than doing this sort of thing.